clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioParsers.scala

```diff
@@ -1,16 +1,16 @@
 package org.broadinstitute.clio.client.commands
 
 import java.net.URI
-import java.nio.file.{Path, Paths}
+import java.time.OffsetDateTime
```

Comment:
How was this working before if this and `java.util.UUID` were not imported?

Reply:
It was imported below, I snuck an optimize-imports in.